sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_call_connection_client.py

```diff
@@ -1138,3 +1139,37 @@ def stop_media_streaming(
             stop_media_streaming_request,
             **kwargs
             )
+        
+    @distributed_trace
+    def interrupt_audio_and_announce(
+        self,
+        target_participant: "CommunicationIdentifier",
+        play_sources: Union[List[Union['FileSource', 'TextSource', 'SsmlSource']]],
```

Comment:
i think playsources should be llist of sourcs, remove the Union it should be look like below        play_sources: List[Union['FileSource', 'TextSource', 'SsmlSource']],